tests/fixtures/common/collections/ansible_collections/testorg/coll_1/plugins/lookup/lookup_1.py

```diff
@@ -1,9 +1,10 @@
-"""
-The get_path lookup plugin
-"""
+"""An ansible test lookup plugin."""
+
 from __future__ import absolute_import, division, print_function
 
+# pylint: disable=invalid-name
 __metaclass__ = type
+# pylint: enable=invalid-name
```

Comment:
This is unnecessary. The previous ignore only applies to the next line.

Reply:
@webknjaz that wasn't my understanding, I thought the scope was file in that case  eg

```# pylint: disable=invalid-name
__metaclass__ = type

__foo__ = type
```
does throw an error but

```

__metaclass__ = type

__foo__ = type
```

throws 2

```
************* Module tests.fixtures.common.collections.ansible_collections.testorg.coll_2.plugins.modules.mod_2
tests/fixtures/common/collections/ansible_collections/testorg/coll_2/plugins/modules/mod_2.py:5:0: C0103: Class name "__metaclass__" doesn't conform to PascalCase naming style (invalid-name)
tests/fixtures/common/collections/ansible_collections/testorg/coll_2/plugins/modules/mod_2.py:7:0: C0103: Class name "__foo__" doesn't conform to PascalCase naming style (invalid-name)
```

Reply:
which has always led me to the assumption, once disabled it was disabled for the file or explicitly re-enabled

Reply:
That's weird. Looks like this may depend on the context and in this case it's global. I guess this is one of the reasons to put the ignore on the same line... Pylint tends to be rather picky with this.

Reply:
Have you checked if the "enable" part really reenables it then? If yes, it's good to merge, better than nothing but still a bit unclear why.

Reply:
@webknjaz I put this together to demonstrate the enable statement, TY again for the review

```
# Demonstration of pylint scope for webnjaz

from pylint import epylint as lint

FILENAME = "code.py"

def check(string: str, count: int):
    with open(FILENAME, 'w') as file:
        file.write(string)
    (pylint_stdout, pylint_stderr) = lint.py_run('code.py', return_std=True)
    stdout = pylint_stdout.getvalue()
    print("=" * 72)
    print(string)
    print(stdout)
    found = stdout.count('invalid-name')
    print(f"Found {found} times, expected {count} times")
    assert found == count, (found, count)


NO_DISABLE = """
__metaclass__ = type
__foo__ = type
"""
check(NO_DISABLE, 2)


DISABLE_ONE = """
# pylint: disable=invalid-name
__metaclass__ = type
# pylint: enable=invalid-name
__foo__ = type
"""
check(DISABLE_ONE, 1)


DISABLE_ALL = """
# pylint: disable=invalid-name
__metaclass__ = type
__foo__ = type
"""
check(DISABLE_ALL, 0)
```

```
(venv) x1 ➜  pylint_test python pylint_demo.py
========================================================================

__metaclass__ = type
__foo__ = type

************* Module code
 code.py:1: convention (C0114, missing-module-docstring, ) Missing module docstring
 code.py:2: convention (C0103, invalid-name, ) Class name "__metaclass__" doesn't conform to PascalCase naming style
 code.py:3: convention (C0103, invalid-name, ) Class name "__foo__" doesn't conform to PascalCase naming style
 
 --------------------------------------------------------------------
 Your code has been rated at -5.00/10 (previous run: 5.00/10, -10.00)
 
 
Found 2 times, expected 2 times
========================================================================

# pylint: disable=invalid-name
__metaclass__ = type
# pylint: enable=invalid-name
__foo__ = type

************* Module code
 code.py:1: convention (C0114, missing-module-docstring, ) Missing module docstring
 code.py:5: convention (C0103, invalid-name, ) Class name "__foo__" doesn't conform to PascalCase naming style
 
 -------------------------------------------------------------------
 Your code has been rated at 0.00/10 (previous run: -5.00/10, +5.00)
 
 
Found 1 times, expected 1 times
========================================================================

# pylint: disable=invalid-name
__metaclass__ = type
__foo__ = type

************* Module code
 code.py:1: convention (C0114, missing-module-docstring, ) Missing module docstring
 
 ------------------------------------------------------------------
 Your code has been rated at 5.00/10 (previous run: 0.00/10, +5.00)
 
 
Found 0 times, expected 0 times
```